src/components/DataView/DataViewActionsRow/DataViewActionsRow.styled.ts

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+export const DataViewActionsRowWrapper = styled.div`
+	align-items: center;
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 20px;
```

Comment:
I believe the margin bottom should be part of .headerRow > .right because if we look at the specs, the whole row should have 20px padding all around